components/datetime/tests/fixtures/tests/components.json

```diff
@@ -26,5 +26,23 @@
                 "indian/en": "Tuesday, Chaitra 21, 1942 Saka at 08:25:07"
             }
         }
+    },
+    {
+        "input": {
+            "value": "2022-05-03T14:15:07.123",
+            "options": {
+                "components": {
+                    "hour": "numeric",
+                    "minute": "numeric",
+                    "second": "two-digit",
+                    "fractional_second": 8
```

Comment:
Suggestion: As stated above, a test like this one with `"fractional_second": 0` would be nice to have, since it's a bit weird and expressible in the type system.